sql/src/main/java/io/crate/analyze/AnalyzedColumnDefinition.java

```diff
@@ -133,6 +139,13 @@ void geoSettings(Settings settings) {
     }
 
     public void dataType(String dataType) {
```

Comment:
it used in create and alter table analyzers, so probably should be a proper place to add the deprecation warning.